feat: mobile tests for GM to channel (#128)
mattermost/mattermost-test-management: data/test-cases/mobile-v2/channels/mobile-test-3.md

---
# (Required) Ensure all values are filled up
name: "Verify opening GM notification after GM is converted to channel by other user"
status: Active
priority: Normal
folder: Channels
authors: "@yasserfaraazkhan"
team_ownership: 
- ICU
priority_p1_to_p4: P2 - Core Functions (Do core functions work?)

# (Optional)
location: null
component: null
tags: []
labels: []
tested_by_contributor: null

# (Optional) Test type and tools
cypress: null
detox: null
mmctl: null
playwright: null
rainforest: []
manual_test_environments: []

# Do not change
id: null
key: null
created_on: null
last_updated: null
case_hashed: null
steps_hashed: null
---

**Step 1**

Test Setup:

> - Have 2 user account to use. eg: User A and User B.

Reviewer: ```suggestion
- Have 2 user accounts to use, e.g. User A and User B.
```

- Login into mobile app with User B.

> - Login into Webapp with User A and create a GM with User B.

Reviewer: ```suggestion
- Login into webapp with User A and create a GM with User B.
```

Reviewer: Should these refer to `DM`s instead of `GM`s since it's just between two users?

Author: @lindalumitchell , i'll add a step to mention using another User acoount to create a GM

- Trigger a mobile notification for User B and verify the app opens up the GM channel.
- With GM channel opened close the app.

> - Convert the GM channel from Web app.

Reviewer: ```suggestion
- Convert the GM channel from webapp.
```

Steps:

> 1. On the mobile.

Reviewer: ```suggestion
1. View on the mobile app.
```
